### ISSUE TITLE
Comprehensive list of impacts of hardware choices on installation time

### ISSUE THREAD

**Issue by G-Guillard** [mediagis/nominatim-docker#436]
The installation of Nominatim requires quite some time, and it is a common request to know how long an installation will take.

Nowadays it is easy to choose from a wide range of hardware on various cloud solutions.  A criterion of choice is obviously the price of the solution, however for the same price there are often a variety of solutions, e.g. CPU-oriented solutions, memory-oriented solutions, storage-oriented solutions, bandwidth-oriented solutions, etc.

Therefore, while it is easy to access outstanding hardware (and to switch later on), it is also easy to choose a suboptimal solution when it comes to Nominatim installation.

It would be very useful if the impact of hardware was documented.  E.g. should one prefer to increase the number of CPUs rather than the amount of RAM at some point (from my experience it seems so, but nowhere did I see it mentioned on the documentation) ?  Is there a point where increasing PostgreSQL memory settings is worthless ?  Etc.

In other words, it would be nice to have a guide of how to choose the right VM hardware settings to install Nominatim the most efficiently.

**Comment by leonardehrenfried** [comment]
We would love such a guide but are not actively working on it.

The best we have is https://github.com/mediagis/nominatim-docker/discussions/265

**Comment by G-Guillard** [comment]
Well, that’s the point of an enhancement request, at some point it might be voted enough so that it becomes a priority. ^^

Thanks for #265, I missed it, useful insights indeed.  #428 is also interesting (indeed I seemed to notice that the installation was greedier on CPUs that on RAM, which I didn’t expect, hence this issue).

Maybe linking those two issues somewhere in the documentation could be a good start to collect feedbacks that could feed such a guide later.

**Comment by leonardehrenfried** [comment]
Voting is not going to help. Since we are all volunteers, we only work on things that _we_ need.
